parser-typechecker/src/Unison/Codebase/Editor/DisplayObject.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE DeriveGeneric #-}
+
+module Unison.Codebase.Editor.DisplayObject where
+
+import Unison.Prelude
+import Unison.ShortHash
+
+data DisplayObject a = BuiltinObject | MissingObject ShortHash | UserObject a
```

Comment:
Moved from `DisplayThing`, which was a dumb name we came up with in a moment of sleep-deprived mania.